.gitignore

```diff
@@ -60,3 +60,7 @@ target/
 
 #Ipython Notebook
 .ipynb_checkpoints
+
+#files
+temp_config.py
```

Comment:
I've added the config file locally as Config.py.  You can/should delete temp_config.py if you are no longer using it.